__tests__/app.test.js

```diff
@@ -3,7 +3,8 @@ const request = require("supertest")
 const app = require("../db/app")
 const db = require("../db/connection")
 const seed = require("../db/seeds/seed")
-const data = require("../db/data/test-data")
+const data = require("../db/data/test-data/index")
```

Comment:
We shouldn't require the data in here to be used in the test assertions. If you need to assert responses that have ids or are the result of joining tables then it won't work. Please write in the tests the shape of the expected responses using Jest matchers etc.